docs/en/observability/synthetics-security-encryption.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[synthetics-security-encryption]]
+= Synthetics Encryption and Security
+
+Elastic Synthetics was designed with security in mind encrypting both persisted and transmitted data.
+This page catalogs the points within the Elastic Synthetics app data is either stored or transmitted in an encrypted fashion.
+
+== Synthetics UI (Kibana App)
+
+Data is stored in Kibana Secure Saved Objects[https://www.elastic.co/guide/en/kibana/master/xpack-security-secure-saved-objects.html], 
+with sensitive fields encrypted. These fields include your script source, params, and global params.
+
+== Synthetics Service
+
+The Global Elastic Synthetics Service performs all communication of sensitive data (both internally, and with Kibana) over encrypted connections
+and encrypts all data persisted to disk as well.
+
+== Synthetics Private Locations
+
+Data is stored in Kibana Secure Saved Objects[https://www.elastic.co/guide/en/kibana/master/xpack-security-secure-saved-objects.html], and encrypted
```

Comment:
I don't know about this one. Does `data` here refer to the integration policies themselves? Sensitive fields otherwise encrypted in our Synthetics UI are stored as plaintext in policies.

Reply:
Yeah just checked. Integration policies are regular saved objects.

Reply:
Hmmm, so any encryption provided would be at the OS level (or CSP infra level), and of course there would be the perms on the object. How restrictive are those?

Reply:
You need `Integrations: Read` and `Fleet: All` permissions to read integration policies.

Reply:
We will be adding the ability to encrypt fields on Agent policies and preventing retrieval from Kibana in the near-future: https://github.com/elastic/ingest-dev/issues/1000